h2o-automl/src/test/java/ai/h2o/automl/AutoMLTest.java

```diff
@@ -31,13 +34,10 @@
 import static junit.framework.TestCase.assertTrue;
 import static org.junit.Assert.*;
 
+@RunWith(H2ORunner.class)
+@CloudSize(1)
 public class AutoMLTest extends water.TestUtil {
-
-  @BeforeClass public static void setup() {
-    stall_till_cloudsize(1);
-//    stall_till_cloudsize(new String[] {"-log_level", "DEBUG"}, 1);
```

Comment:
Log has a method `setLogLevel` - in case we want to preserve this

Reply:
Unused comment.

Reply:
ouch :)